packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -861,6 +861,21 @@ const schema: SchemaType<DbComment> = {
     canCreate: ['admins'],
     canUpdate: [userOwns, 'admins'],
   },
+
+  originalDialogueId: {
+    ...foreignKeyField({
+      idFieldName: "originalDialogueId",
+      resolverName: "originalDialogue",
+      collectionName: "Posts",
+      type: "Post",
+      nullable: true,
+    }),
+    optional: true,
+    hidden: true,
+    canRead: ['guests'],
+    canUpdate: ['sunshineRegiment', 'admins'],
+    canCreate: ['members', 'sunshineRegiment', 'admins'],
```

Comment:
The top-level field should also explicitly have `nullable: true` set.  (The `nullable: true` inside of the `foreignKeyField` args is for whether the generated resolver field is nullable, and doesn't affect whether the actual database column generated by this will be nullable, nor the corresponding type annotations.)

Reply:
Unfortunately you will probably need to restore your migration files and do that bit from scratch, since it'll cause a change in the generated schema hash